itest/litd_custom_channels_test.go

```diff
@@ -456,14 +457,44 @@ func testCustomChannels(_ context.Context, net *NetworkHarness,
 		false,
 	)
 
-	// For some reason, the channel funding output of the immediately closed
-	// channel is still present in the asset DB, even after we import the
```

Comment:
Is this fixed by the fix to `SetAssetSpent` in tapd? Or I guess it's the responder co-op close issue? They can't insert the spend as they don't know of the funding output/input.

Reply:
It's fixed by the SetSpend fix.